draft-ietf-quic-multipath.md

```diff
@@ -889,10 +892,6 @@ Client                                                      Server
 ~~~
 {: #fig-example-path-close1 title="Example of closing a path."}
 
-After a path is abandoned, the path identifier associated with the path
-is considered retired and MUST NOT be reused in new paths for security
-considerations discussed in {{multipath-aead}}.
-
```

Comment:
We should also move the next paragraph.

Reply:
This is moved as part of PR #354. I moved it into the MAX_PATHS section. Maybe it should rather go above into the path closure section but I think there is actually more that needs moving.